packages/web-components/src/components/rux-checkbox/rux-checkbox.tsx

```diff
@@ -168,7 +172,7 @@ export class RuxCheckbox implements FormFieldInterface {
                             onBlur={this._onBlur}
                             ref={(el) => (this._inputEl = el)}
                         />
-                        <label htmlFor={checkboxId}>
+                        <label htmlFor={checkboxId} part="label">
```

Comment:
@markacianfrani here is an example of adding a label part even though there is a slot associated with it. We're rending the label text, so I figured most the time a slot wouldn't be used and we should provide a part.